File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { VisitorFrequenciesLucerneChSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await VisitorFrequenciesLucerneChSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("visitor-frequencies-lucerne-ch_sdk")

describe("VisitorFrequenciesLucerneChSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
